_episodes/01-starting-with-data.md

```diff
@@ -319,6 +319,20 @@ Let's look at the data using these.
 
 ## Calculating Statistics From Data In A Pandas DataFrame
 
+We've read our data into Python. Now, we want to understand better the data
+we are dealing with, and for that we can perform a basic summary of statistics
+for each column in our data set. The **pandas function `describe`** will return 
+descriptive stats including: mean, median, max, min, std and count for each column 
+in the data. 
+
+```python
+# Summary of statistics for each column of the data set
+surveys_df.describe()
+```
+Pandas' `describe` function will only return summary values for columns
+containing numeric data (note that for the columns `species_id` and `sex`
+nothing was returned). Further exploring our data set, we might want
+to know how many animals were collected in each plot, or how many of each.
```

Comment:
This whole new section partially repeats the text that follows.
Please either remove the duplicate text here or in the paragraphs that follow.
Also, please use the following syntax for code blocks
```markdown
~~~
python code
~~~
{: .language-python}
```

Reply:
I cloned the repository to my PC, made the changes you requested and pushed it back @maxim-belkin, I hope to have done it correctly, if not please let me know where I did wrong and I will fix it.

Reply:
@gustavothomas17 apologies for the delay...

You used your `gh-pages` branch which complicates things a little bit. I will close this PR, but please submit a new one with your suggested changes using a different branch. Here is what you should do:

```
git clone https://github.com/datacarpentry/python-ecology-lesson new-directory
cd new-directory
git branch my-changes
git checkout my-changes
## implement your changes
git add -u
git commit
git push --force https://github.com/gustavothomas17/python-ecology-lesson gh-pages:gh-pages
git push https://github.com/gustavothomas17/python-ecology-lesson
```
and then submit this PR again.

I will add a few comments to your PR - please take them into account when submitting a new request.

Reply:
Thank you very much for your time @maxim-belkin, I hope to have followed your instructions properly this time.